routes/archive.js

```diff
@@ -207,3 +210,44 @@ exports.view_latest = function(req, res){
   var contentData = campaignsCache[0];
   res.render('index', {campaigns: campaignsCache, issue: contentData});
 };
+
+exports.search = function (req, res) {
+  let searchResult = {success: false, message: "No results..."};
+
+  if (req.query.query) {
+    let offset = (1 * req.query.offset) || 0;
+
+    axios.get(CLOUD_SEARCH_URL, {
+      params: {
+        "q": req.query.query,
+        "size": 25,
+        "start": offset,
+        "return": "_all_fields",
+        "sort": "custom_score desc"
+      }
+    }).then(r => {
+      if (r.data.hits && r.data.hits.found) {
+        searchResult.success = true;
+        searchResult.results = r.data.hits.hit;
+        if (offset !== 0) {
+          searchResult.previousUrl = buildSearchUrl(req.query.query, offset - PAGE_SIZE);
+        }
+        if (offset + PAGE_SIZE < r.data.hits.found) {
```

Comment:
czyli na każdej kolejnej stronie pobieramy wyniki od 0 do PAGE_SIZE*numer strony?

Reply:
a nie bo `hits` to jest liczba wyników wszystkich, już rozumiem :)

Reply:
Pobieramy po 25 rekordow (parametr size) i przesuwamy sie uzywajac offsetu (parametr start)